testing/src/main/java/org/oppia/android/testing/TextInputActionTestActivity.kt

```diff
@@ -0,0 +1,21 @@
+package org.oppia.android.testing
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+
+/** Test Activity for TextInputActionTest. */
+class TextInputActionTestActivity : AppCompatActivity() {
+  /** Sets theme for the [TextInputActionTestActivity] to AppCompat_Light when it is created. */
```

Comment:
Can be omitted (this is inherent from reading the code, plus we generally don't document overrides since the documentation should be the same as the base method as part of overriding the method's contract).

Reply:
Done.